descriptors/ModuleDescriptor-template.json

```diff
@@ -1,11 +1,95 @@
 {
-  "id": "${artifactId}-${version}",
-  "name": "",
+  "id": "@artifactId@-@version@",
+  "name": "Remote storages API module",
   "requires": [],
-  "provides": [],
-  "permissionSets": [],
+  "provides": [
+    {
+      "id": "remote-storage-configurations",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/remote-storages/configurations",
+          "permissionsRequired" : [ "remote-storage.configuration.item.post" ]
```

Comment:
Mismatching in names remote-storage and remote-storages. See https://github.com/folio-org/mod-remote-storage/pull/3/files#r544261452. Is it OK? CC @MikhailFokanov